recipes/qgrid/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "e885cd1b2138720850c0da1146f06d69343cd2fdab10bb9ce3337fb23adaebeb" %}
+
+package:
+  name: qgrid
+  version: {{ version }}
+
+source:
+  fn: qgrid-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/q/qgrid/qgrid-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Minor comment: Would use 2 space indenting here as it was used elsewhere in the recipe.